package.json

```diff
@@ -15,7 +15,6 @@
   "scripts": {
     "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
     "cover:test": "nyc --reporter=lcov --reporter=text-lcov",
-    "bower": "node node_modules/gulp-cli/bin/gulp install",
```

Comment:
The script was initially called `postinstall` , but I changed it to bower when I was trying to integrate the project with coveralls because travis was running the postinstall script.
But now we have reverted to `postinstall`.